tests/test_pipelines_feature_extraction.py

```diff
@@ -74,10 +75,8 @@ def get_test_pipeline(self, model, tokenizer, feature_extractor):
             self.skipTest("No tokenizer")
             return
 
-        elif isinstance(model.config, (LxmertConfig, CLIPConfig, Wav2Vec2Config)):
-            self.skipTest(
-                "This is an Lxmert bimodal model, we need to find a more consistent way to switch on those models."
-            )
+        elif isinstance(model.config, (LxmertConfig, CLIPConfig, Wav2Vec2Config, HubertConfig)):
```

Comment:
```suggestion
        elif FEATURE_EXTRACTOR_MAPPING.get(model.config, None) is not None:
```

maybe make it more general here?

Reply:
Yes !